src/fides/api/ctl/migrations/versions/d65bbc647083_adds_gpc_info_to_consent_table.py

```diff
@@ -5,30 +5,39 @@
 Create Date: 2023-02-15 13:57:36.159161
 
 """
-from alembic import op
 import sqlalchemy as sa
-
+from alembic import op
```

Comment:
Is this file auto-formatting collateral?

Reply:
yes